schemas/src/main/java/com/scottlogic/deg/schemas/common/ValidationResult.java

```diff
@@ -5,10 +5,14 @@
 public class ValidationResult {
     public final List<String> errorMessages;
 
-    public ValidationResult(List<String> errorMessages){
+    public ValidationResult(List<String> errorMessages) {
         this.errorMessages = errorMessages;
     }
 
+    public void addErrorMessages(List<String> errorMessages) {
```

Comment:
This could be removed given the above comments.

Reply:
removed